tasks/kubectl-run-acceptance-tests.yml

```diff
@@ -37,6 +37,6 @@ run:
       kubectl wait --for=condition=Ready pod/acceptance-tests --timeout=200s
 
       kubectl exec -it acceptance-tests -- /bin/bash -c \
-      "sleep 2; behave acceptance_tests/features --no-skipped --no-logcapture --logging-level WARN"
+      "sleep 2; behave acceptance_tests/features --no-logcapture --logging-level WARN  --tags=~@regression"
```

Comment:
I think we should keep `--no-skipped`, otherwise behave logs all the steps it's skipping.